ui-v2/ember-cli-build.js

```diff
@@ -5,7 +5,8 @@ module.exports = function(defaults) {
   const env = EmberApp.env();
   const prodlike = ['production', 'staging'];
   const isProd = env === 'production';
-  const isProdLike = prodlike.indexOf(env) > -1;
+  // leave this in for now for when I start a proper staging env
+  // const isProdLike = prodlike.indexOf(env) > -1;
```

Comment:
A couple opinions on this, take them or leave them. 1) Saying things like "when I start" in code comes off kind of controlling. Am I supposed to do a git blame to see who "I" is? Should I leave this alone and let "I" deal with it? 2) It seems easy enough to add this line back once there is a staging env rather than having an unused wart for a possible future.

Reply:
No prob,I tweaked the comment in https://github.com/hashicorp/consul/pull/5021 to reword it slightly.

You don't have to, but to explain a little - if you look at the blame for this file:

https://github.com/hashicorp/consul/blame/88027a940b088d4e3c96b8dbcd9e557c8d524881/ui-v2/ember-cli-build.js

You'll see I did it during a 'Comment clean up' commit. I didn't remove this unused code on purpose, but added the comment to explain why I didn't remove it like I did with others. For me it will be useful at somepoint, maybe for others also. It's in 'node-world' so it doesn't end up in shipping code (and is a comment anyway), so I've chosen to leave it in for the moment.

We are currently looking at setting up a more permanent staging environment for the UI, so this might come in useful 🔜